### ISSUE TITLE
Right click menu on multiple monitors (Windows 10)

### ISSUE THREAD

**Issue by iamtesch** [HebiRobotics/scope2-feedback#37]
When I right click a module, text field, or other on Windows 10, the right click menu appears at a completely different location than my cursor.

This issue also existed in the pre-redesign scope, and I've added screenshots/etc there.

**Comment by ennerf** [comment]
Yes, I still have never been able to reproduce it. Are you running multiple monitors with different resolutions and DPI settings? Mine are always the same, so that might be where the issue is coming from.
